.travis.yml

```diff
@@ -1,16 +1,11 @@
 language: php
 
 php:
-    - 5.6
-    - 7.0
```

Comment:
You removed the 7.0 tests... but composer.json says php >= 7.0

Reply:
good shot!